### ISSUE TITLE
Inverse Problem

### ISSUE THREAD

**Issue by RoshenS** [RoshenS/PINNs#5]
Using the physics loss of a system, a tunable parameter, and samples from a physical system, estimate an unknown parameter.

1. Change the loss function
2. Incorporate a tunable parameter
3. Estimate the unknown parameter after training

**Comment by RoshenS** [comment]
The approximation was within 2% of the actual value. Lowering the learning rate and training the model for longer may obtain more reliable convergence.
